lib/liquid/translators/tags/custom_block.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Liquid.Translators.Tags.CustomBlock do
+  alias Liquid.Translators.General
+  alias Liquid.{Template, Block}
+
+  def translate(custom_name: name, custom_markup: markup, body: body) do
```

Comment:
Can you add @doc for public functions and @specs for all functions?

Reply:
done